Python/ceval.c

```diff
@@ -1756,6 +1756,7 @@ _PyEval_EvalFrameDefault(PyFrameObject *f, int throwflag)
 
             if (iter == NULL) {
                 format_awaitable_error(Py_TYPE(iterable),
+                                       _Py_OPCODE(next_instr[-3]), 
```

Comment:
>_Py_OPCODE(next_instr[-3]) [](start = 39, length = 26)

seems like this should have a bounds check, e.g.:

async def f():
    await 1

Results in:

0 LOAD_CONST               1 (1)
2 GET_AWAITABLE

Which is going to index into random memory here.